w03h03/test/pgdp/SeamCarvingTest.java

```diff
@@ -0,0 +1,85 @@
+package pgdp.image;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SeamCarvingTest {
+
+    @DisplayName("Testing computeGradientMagnnitude method.")
+    @Test
+    public void computeGradientMagnitudeTest() {
+        var output = new SeamCarving();
+        int v1 = -10799838;
+        int v2 = -9217977;
+        Assertions.assertEquals(3170, output.computeGradientMagnitude(v1, v2));
+    }
+
+    @DisplayName("Testing toGradientMagnitude method")
+    @Test
+    public void toGradientMagnitudeTest() {
+        var output = new SeamCarving();
+            int[] testPicture = new int[]{-10141908, -10470873, -10799838, -10536921, -9944784, -9155268, -9418440, -9549771, -9812686, -10272724, -10534869, -10008008, -9217977, -9282739, -10269377, -11585494, -12506339, -12111581, -11255501, -11058122, -11387341, -11322316, -11782867, -12374234, -12505822, -12439516, -12176344, -12176344, -12241626, -12110041, -11846869, -11452881, -11255502, -11124432, -11322324, -11454678, -11652057, -11586777, -11586777, -11653083};
+            int[] gradientMagnitude = new int[40];
+            int[] expectedGradientMagnitude = new int[] {2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 917, 742, 2193, 1353, 1883, 5492, 2147483647, 2147483647, 2738, 1756, 1842, 4514, 4742, 1921, 2147483647, 2147483647, 760, 123, 355, 271, 363, 262, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647, 2147483647};
+        output.toGradientMagnitude(testPicture, gradientMagnitude, 8, 5);
+        assertArrayEquals(expectedGradientMagnitude, gradientMagnitude, "Gradient Magnitude not correct!");
+        }
```

Comment:
Is this file formatted with your IDE? You can just press the shortcut for formatting files :+1: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_file

Reply:
Habe die Datei formatiert und committed. Passt das so? Vim hat mir das Format irgendwie zerschossen gehabt.🤷‍♂️

Reply:
Yes, looks better - thanks! ❤️